numhask/src/NumHask/Algebra/Metric.hs

```diff
@@ -306,12 +307,13 @@ instance Metric Word64 Word64 where
   distanceLp p a b = fromInteger (normLp (toInteger p) (toInteger a - toInteger b))
 
 -- | todo: This should probably be split off into some sort of alternative Equality logic, but to what end?
-class (Eq a, AdditiveGroup a) =>
+class (Eq a, AdditiveUnital a) =>
       Epsilon a where
   nearZero :: a -> Bool
   nearZero a = a == zero
 
   aboutEqual :: a -> a -> Bool
+  default aboutEqual :: AdditiveGroup a => a -> a -> Bool
```

Comment:
@tonyday567 that's what I meant in the ticket. With this formulation, I am able to write an `Epsilon` instance for `Logfield`